.github/workflows/build-server.yaml

```
@@ -130,56 +130,92 @@ jobs:
           # append to .env
           echo -e "\nSENTRY_AUTH_TOKEN=$SENTRY_AUTH" >> desci-server/.env
 
-      - name: Build and tag the image (DEV)
+      - name: Build and push image (DEV) - With Registry Caching
         if: github.ref == 'refs/heads/develop'
         run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:2:37: Double quote to prevent globbing and word splitting

(shellcheck)